src/components/RegionItem/HomepageItems.tsx

```diff
@@ -1,19 +1,75 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import RegionItem from './RegionItem';
-// import RegionItemSkeleton from './RegionItemSkeleton';
-import regions from 'common/regions';
+import RegionItemSkeleton from './RegionItemSkeleton';
 import { RegionItemsWrapper } from 'screens/HomePage/New/NewHomepage.style';
+import { useGeolocation } from 'common/hooks';
+import {
+  getGeolocatedRegions,
+  GeolocatedRegions,
+  Region,
+} from 'common/regions';
+
+enum ItemsState {
+  LOADING,
+  READY,
+  NOT_FOUND,
+}
 
 const HomepageItems: React.FC = () => {
-  const stateRegion = regions.findByFipsCodeStrict('23005');
-  const countyRegion = regions.findByFipsCodeStrict('38860');
+  const { geolocationData, isLoading } = useGeolocation();
```

Comment:
`useGeolocation` is also called a level up in `NewHomepage.tsx`. want to pass down `geolocationData` and `isLoading` from there?